src/misc/programs.cpp

```diff
@@ -775,6 +780,7 @@ void CONFIG::Run(void)
 					         sec_name);
 					return;
 				}
+				// XXX
```

Comment:
?

Reply:
Oops, missing implementation. The change above makes `config -get window_titlebar` display sane values if custom program name contains national characters. The missing implementation here should handle `config -get sdl window_titlebar` command. Fixed.

Reply:
Glad to hear our code reviews serve their intended purpose 😄